resources/css/progress.css

```diff
@@ -59,32 +26,29 @@
 }
 
 .completion-icon {
-  position: absolute;
-  top: -6.25px;
-  right: 0;
   display: flex;
   justify-content: center;
   align-items: center;
-  height: 19px;
-  width: 19px;
+  height: 20px;
+  width: 20px;
   border: solid 2px;
   border-radius: 50%;
-  font-family: emoji;
-  font-size: 7pt;
-  cursor: default;
+  font-size: 10px;
+  line-height: 1em;
+  margin-left: -1px;
```

Comment:
Just realized that the border radius was already removed, nice :) 
Removed the gap by removing the border and moving the completion icon one pixel to the left.

Reply:
Looks like the negative margin was only needed when zoomed in (which also is the cause of the apparently missing bottom left radius in your screenshot i assume?)

Reply:
>(which also is the cause of the apparently missing bottom left radius in your screenshot i assume?)

Now that you mention it... you're right. They constantly appear off to me. Zooming in seemingly messes the rendering for corners.